src/Miew.js

```diff
@@ -3749,6 +3790,37 @@ Miew.prototype.scale = function (factor) {
   this._needRender = true;
 };
 
+/**
+ * Center view on selection
+ * @param {subset | string} selector - translation value (Ang) along model's X axis
+ */
+Miew.prototype.center = function (selector) {
+  if (selector === undefined) { // on current sellection
+    this.setPivotSubset();
+    this._needRender = true;
+    return;
+  }
+  if (selector.obj !== undefined && ('atom' in selector.obj || 'residue' in selector.obj)) { // from event with selection
+    if ('atom' in selector.obj) {
+      this.setPivotAtom(selector.obj.atom);
+    } else {
+      this.setPivotResidue(selector.obj.residue);
+    }
+    this._needRender = true;
+    return;
+  }
+  if (selector.obj === undefined && selector !== '') {
+    const sel = selectors.parse(selector);
+    if (sel.error === undefined) {
+      this.setPivotSubset(sel);
+      this._needRender = true;
+      return;
+    }
+  }
+  this.resetPivot();
+  this._needRender = true;
+};
+
```

Comment:
Additionally, it will be nice to add comments on the processing of three argument types:
- empty 
- string 
- selector

Reply:
Thank you
Add more comments which are more detailed